src/utils/getRandomNum.js

```diff
@@ -0,0 +1,8 @@
+import { MissionUtils } from '@woowacourse/mission-utils';
+import { MIN_RANDOM_NUM, MAX_RANDOM_NUM } from '../constants/constants.js';
+
+const getRandomNum = function getRandomNumFunc() {
+  return MissionUtils.Random.pickNumberInRange(MIN_RANDOM_NUM, MAX_RANDOM_NUM);
+};
```

Comment:
```suggestion
const getRandomNum = function getRandomNumFunc(minRandomNumber, maxRandomNumber) {
  return MissionUtils.Random.pickNumberInRange(minRandomNumber, maxRandomNumber);
};
```
인자로 랜덤 값의 범위를 받으면 요구사항이 변경했을 때도 유연하게 동작할 수 있을 것 같은데 어떻게 생각하시나요?

Reply:
해당 미션에서는 0부터 9사이라는 범위가 지정되었기 때문에 상수로 0과 9를 지정해주는 것이 더욱 좋다고 생각합니다! 만약 랜덤으로 범위를 받는 문제라면 인자로 받는 것도 좋은 방법일 것 같아요!

Reply:
약간 위에 리뷰랑은 반대되는 의견인것 같네요🤔

위에서는 추상화를 했을 때 변경에 유연하고, 함수 네이밍으로 통해 동작을 유추할 수 있기 때문에 올바른 추상화였다고 말씀해주셨는데 여기서는 미션의 요구사항이 안바뀔거라고 단정되어 있어 있는 느낌이 들었는데 혹시 그렇게 생각하신 이유가 있으신가요??

Reply:
좋은 질문 감사합니다! 😊 앞서 설명드린 것처럼 splitByComma를 추상화한 이유는 코드의 의도를 더욱 명확히 전달하고 유지보수에 유리하게 하기 위함이었습니다. 특히 함수 이름을 통해 기능의 목적이 잘 드러나도록 하여 가독성을 높이려는 의도였습니다!

getRandomNum 함수는 미션 요구사항에서 0과 9라는 고정된 범위가 명시되어 있었기 때문에 인자를 사용하지 않고 상수로 고정해 불필요한 코드 복잡성을 줄이고자 했습니다. 명확한 고정값의 경우 상수로 관리하는 것이 코드의 의도를 직관적으로 드러내기 때문에 이해하기 더 쉽다고 판단했습니다. 요구사항이 바뀌지 않는다고 단정하기보다는 주어진 고정값은 상수로 관리하는 것이 바람직하다고 생각하여 상황에 따라 방식을 다르게 선택하게 되었습니다. 추가적인 의견이나 피드백이 있으시면 언제든 말씀해 주세요!

Reply:
잘 설명해주셔서 바로 이해할 수 있었던 것 같네요!

요구사항이 바뀌면 해당 상수 값을 변경하면 되는 것은 동의하지만, `getRandomNum` 함수는 충분히 범용적으로 사용될 수 있다고 생각이 들었던 것 같아요🤔

만약 두 곳에서 사용하는데 한 곳은 범위가 `1~9`이고 나머지 한 곳은 `1~100`인 경우에는 고정된 상수 값을 변경해도 해결할 수 없다고 생각해서 인자로 주입하는 방식이 조금 더 유연하다고 생각했던 것 같네요!

말씀해주신 의도는 명확히 이해할 수 있었던 것 같습니다 감사합니다👍👍👍